examples/platform_services/android/app/src/main/java/com/example/flutter/ExampleActivity.java

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package com.example.flutter;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationManager;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import io.flutter.view.FlutterMain;
+import io.flutter.view.FlutterView;
+
+import java.io.File;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class ExampleActivity extends Activity {
+    private static final String TAG = "ExampleActivity";
+
+    private FlutterView flutterView;
```

Comment:
FlutterActivity already exposes the FlutterView:

https://docs.flutter.io/javadoc/io/flutter/app/FlutterActivity.html#getFlutterView()

IMHO, we should clean this up before landing this example.